mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java

```diff
@@ -104,4 +122,73 @@ protected void initializeMailboxManager() throws Exception {
         storeMailboxManager.setMessageSearchIndex(messageSearchIndex);
         storeMailboxManager.init();
     }
+
+    @Ignore("MAILBOX-301 Lucene terms length exceeded on some emails")
+    @Test
+    public void termsBetweenElasticSearchAndLuceneLimitDueTuNonAsciiCharsShouldNotMakeIndexingFail() throws Exception {
```

Comment:
Also https://www.elastic.co/guide/en/elasticsearch/guide/current/multi-fields.html is a tutorial that show how to use mutlifiield properly for sorting, and this is not what we do here. We analyze the text which is by the way stupid (I will explain that in the mail that I will send about my audit on ES mapping). I insist you read this also: https://www.elastic.co/guide/en/elasticsearch/reference/2.2/multi-fields.html